lk2nd/device/dts/msm8952/msm8937-xiaomi-land.dts

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
```

I'd prefer bsd3 for new dt submissions, like what linux prefers nowdays. (for all files here)